pkg/timeutils/timezones.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package timeutils
+
+import (
+	"io/ioutil"
+	"sort"
+	"strings"
+	"unicode"
+)
+
+// timezones is a slice of valid timezones.
+var timezones []string
+
+// usTimezones is the list of US timezones.
+var usTimezones []string
+
+// isAlpha returns true if all characters in the string are letters.
+func isAlpha(s string) bool {
```

Comment:
`alpha` makes it sound like we're asking if it's a greek letter, maybe `allLetters` or something?